packages/x-scheduler/src/primitives/time-grid/event-resize-handler/TimeGridEventResizeHandler.tsx

```diff
@@ -41,20 +42,17 @@ export const TimeGridEventResizeHandler = React.forwardRef(function TimeGridEven
     return draggable({
       element: domElement,
       getInitialData: ({ input }) => ({
+        ...getSharedDragData(input),
         source: 'TimeGridEventResizeHandler',
-        id: eventId,
-        start: eventStart,
-        end: eventEnd,
         side,
-        position: getCursorPositionRelativeToElement({ ref, input }),
       }),
       onGenerateDragPreview: ({ nativeSetDragImage }) => {
         disableNativeDragPreview({ nativeSetDragImage });
       },
       onDragStart: () => setIsResizing(true),
       onDrop: () => setIsResizing(false),
     });
-  }, [eventStart, eventEnd, eventId, side, setIsResizing]);
+  }, [side, setIsResizing, getCursorPositionInElementMs, getSharedDragData]);
```

Comment:
```suggestion
  }, [side, setIsResizing, getSharedDragData]);
```
I think we don't need `getCursorPositionInElementMs` here 👀